### ISSUE TITLE
call force dequeue after compute

### ISSUE THREAD

**Issue by rnugent3** [HydrologicEngineeringCenter/fda-model#244]
this is a little tricky because we need to call this at the histogram level so the right place is not obvious

**Comment by HenryGeorgist** [comment]
https://github.com/HydrologicEngineeringCenter/fda-model/blob/2298b2450b565012de0c4cdfd4acd2408b6c1130/fda-model/compute/ImpactAreaScenarioSimulation.cs#L249
after line 249 in the current compute

**Comment by rnugent3** [comment]
I thought so, but it is not as straightforward as calling ForceDeQueue. We'd have to write a ForceDeQueue method on ImpactAreaSimulationResults to force dequeue all histograms in impact area simulation results. There are potentially very many.

**Comment by HenryGeorgist** [comment]
right. similar to the check for convergence which has to loop through a number of results(or at least it did at one point). follow that pattern.
